pkg/minikube/config/config.go

```diff
@@ -48,6 +49,10 @@ const (
 	ShowDriverDeprecationNotification = "ShowDriverDeprecationNotification"
 	// ShowBootstrapperDeprecationNotification is the key for ShowBootstrapperDeprecationNotification
 	ShowBootstrapperDeprecationNotification = "ShowBootstrapperDeprecationNotification"
+	// TimeFormat is the format that should be used when outputing time
+	TimeFormat = time.RFC1123
```

Comment:
I don't understand what the time output has to do with config files?

I suspect this constant is in the wrong place.

Reply:
Moved to consts file